PusherClient/EventEmitter.cs

```diff
@@ -16,7 +16,9 @@ public void Bind(string eventName, Action<dynamic> listener)
         {
             if(_eventListeners.ContainsKey(eventName))
             {
-                _eventListeners[eventName].Add(listener);
+                // To avoid triggering the same bind multiple times

```

Comment:
I wouldn't expect you to need to bind to events on a channel after subscription. You should be able to:

```cs
var pusher = new Pusher(...);
pusher.Connect();

// ... wait for connection (although you shouldn't really need to have to do that - but we'll keep that for another day)

var chatChannel = pusher.Subscribe("chat");
chatChannel.Bind("my-message", (dynamic data) =>
    {
        Console.WriteLine("[" + data.name + "] " + data.message);
    });
```

So, to me it seems the [code in the ExampleApplication](https://github.com/pusher-community/pusher-websocket-dotnet/blob/master/ExampleApplication/Program.cs#L101-L104) is causing confusion and is doing something a bit strange.

**hands up** - it has been a while since I've looked at this library. Hopefully I'll have a chance to get back up to speed as part of looking at this pull request.